guides/v2.1/cloud/setup/first-time-setup-import-import.md

```diff
@@ -36,6 +36,7 @@ Before you continue, make sure you know the SSH or HTTPS {% glossarytooltip a05c
 To create a remote Git reference:
 
 1.  Log in to your local {{site.data.var.ece}} development machine as, or switch to, the [Magento file system owner]({{ page.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+
 1.  Make a copy of `composer.json` _in a non-tracked directory_ so it doesn't get overwritten.
```

Comment:
This is out of the scope of your edits, but this step bugs me a little. "Make a copy" is weird and the inclusion of "in a" inside the emphasis is weird. I would consider changing this to:
1.  Copy the `composer.json` file to a _non-tracked_ directory to prevent an accidental overwrite.

Reply:
Yeah--there are many editorial issues on this page. I tried to restrain myself.